README.md

```diff
@@ -62,6 +62,71 @@ def chat_demo():
 
 `https://<your_app_name>.skygeario.com/static/demo/index.html`
 
+### Sending push notification
+
+Push notification can be implemented by the following code.
+Please ensure APNS certificate and private key are properly setup,
+if you are using [Skygear.io], you can configure it at the setting panel.
```

Comment:
in the Settings panel. (and I am not sure if that is called a panel, but I could go with that)